docs/built_in_extensions.rst

```diff
@@ -108,6 +108,37 @@ The HTTP entrypoint is built on top of `werkzeug <http://werkzeug.pocoo.org/>`_.
     Date: Fri, 13 Feb 2015 14:58:48 GMT
 
 
+Expected exceptions can be added to ``expected_exceptions``. This will result in 400 HTTP error to be returned to the client. You can also change exception's formatting by overriding :meth:`~nameko.web.HttpRequestHandler.response_from_exception`:
```

Comment:
This is a bit self-referential somewhere describing why an exception might be "expected" at all (the name is a bit of an oxymoron). As discussed below I think we should have a separate section of the docs for this, and reference it from here. I am happy to add that as a separate PR.